sdk/core/core-tracing/lib/tracerProxy.ts

```diff
@@ -23,5 +28,8 @@ export function getTracer() {
   if (!cache.tracer) {
     cache.tracer = new NoOpTracer();
   }
+  if (env.AZURE_TRACING_DISABLED) {
```

Comment:
it does feel weird that this doesn't have to be truthy at all, e.g. someone could set AZURE_TRACING_DISABLED=false or AZURE_TRACING_DISABLED=0 and it would still disable